src/resources/resources.json

```diff
@@ -3759,26 +3763,26 @@
   },
   "ratingDialog": {
     "title": {
-      "en": "Rate document",
-      "de": "Dokument bewerten"
+      "en": "Rating details",
+      "de": "Bewertungsdetails"
     },
-    "noExistingRatingExplanationMarkdown": {
-      "en": "You have not submitted a rating for this document.\n\nYou can rate the document by clicking on the stars below.\n",
-      "de": "Du hast keine Bewertung für dieses Dokument abgegeben.\n\nBewerte das Dokument, indem du auf die Sterne unten klickst.\n"
+    "stars": {
+      "en": "{starCount, plural, =1 {# star} other {# stars}}",
+      "de": "{starCount, plural, =1 {# Stern} other {# Sterne}}"
     },
-    "existingRatingExplanationMarkdown": {
-      "en": "You have rated this document on {ratingDate} with **{ratingValue, plural, =1 {# star} other {# stars}}**.\n\nYou can change or delete your rating by clicking on the stars or the delete icon below.\n",
-      "de": "Du hast dieses Dokument am {ratingDate} mit **{ratingValue, plural, =1 {# Stern} other {# Sternen}}** bewertet.\n\nDu kannst deine Bewertung ändern oder löschen, indem du auf die Sterne oder das Löschen-Symbol unten klickst.\n"
+    "totalRatingHeader": {
+      "en": "Total rating",
```

Comment:
Should we say "Average rating" instead of "total"? 🤔

Reply:
Or "Global average rating", if you feel like it needs to be more clear that it contains all ratings?

Reply:
Changed it to "Average rating" / "Durchschnittliche Bewertung".

Reply:
👍